migrate.py

```diff
@@ -1333,6 +1333,13 @@ def assemble_collections(checkout_path, spec, args, target_github_org):
                         # TODO: handle powershell import rewrites
                         shutil.copyfile(src, dest)
                         continue
+                    elif plugin_type in ('modules',) and src.endswith('__init__.py'):
```

Comment:
No need for a tuple unless you have more choices, also be more strict about the file name and not match some `safsdf__init__.py`:
```suggestion
                    elif plugin_type == 'modules' and os.path.basename(src) == '__init__.py':
```